examples/ads.amp.html

```diff
@@ -386,19 +386,19 @@ <h2 id="atomx">Atomx</h2>
   
   <h2 id="caajainfeed">CA A.J.A. Infeed</h2>
   <amp-ad width="320" height="120"
-          type="caajainfeed"
-          data-ad-spot="thtuegn1pAQ"
-          data-test="true">
-      <div placeholder></div>
-      <div fallback></div>
+      type="caajainfeed"
+      data-ad-spot="thtuegn1pAQ"
+      data-test="true">
+    <div placeholder></div>
+    <div fallback></div>
   </amp-ad>
 
   <h2 id="caprofitx">CA ProFit-X</h2>
   <amp-ad
-    width="320"
-    height="50"
-    type="caprofitx"
-    data-tagid="17359">
+      width="320"
```

Comment:
since you fix the indention. What about also unify the style here. put `width` and `height` in the same line.